appyters/RNA_Level_Functional_Predictions/static/AutocompleteField.js

```diff
@@ -0,0 +1,123 @@
+// Adapted from W3schools
+
+function runall(name, file_path){
+    
+function readJSONFromURL(file_path) {
+    $.getJSON(file_path, function(data) {
```

Comment:
`$` is not guaranteed to be instantiated at this point, you can either use requirejs to ensure it is (as done in `form.j2`) or use requirejs to declare your module import thus guaranteeing it will be present. See: https://requirejs.org/docs/api.html#defdep  or you can import your file with requirejs and specify jquery as a dependency in the shim https://requirejs.org/docs/api.html#config-shim 

But fortunately since everything is in a function, a simple fix for this that doesn't necessitate additional requirejs usage is to pass $ in your runall:
```
function runall($, name, file_path)
```

and include it at the call-site where it *is* defined with requirejs.